manifests/init.pp

```diff
@@ -196,6 +196,7 @@
   $passenger_package_ensure                                  = 'present',
   String[1] $passenger_package_name                          = $nginx::params::passenger_package_name,
   Optional[Stdlib::HTTPUrl] $repo_source                     = undef,
+  Optional[Float] $dnfmodule                                 = undef,
```

Comment:
I would not use a float to represent a package version… You know:

```puppet
0.1 * 0.2 #=> 0.020000000000000004
```

Reply:
I have changed from Float to String.